packages/vue-components/src/panels/NestedPanel.vue

```diff
@@ -236,10 +248,19 @@ export default {
 
     .caret-wrapper {
         float: left;
-        display: inline-block;
         width: 32px;
     }
 
+    .minimal-caret-wrapper {
+        display: inline-block;
+        font-size: 13px;
+        margin-right: 2px;
+    }
+
+    .glyphicon {
+        display: inline-block;
+    }
```

Comment:
I believe the provided css from bootstrap-glyphicons already set the display to inline-block so should be able to remove this.